fcrepo-http-api/src/main/java/org/fcrepo/http/api/FedoraLdp.java

```diff
@@ -763,6 +763,23 @@ private String mintNewPid(final String slug) {
         return pid;
     }
 
+    /**
+     * perform save with checking to prevent SN-Sibling nodes from creating
+     * @param resource the FedoraResource need to save
+     * @throws RepositoryException
+     */
+    private void save(final FedoraResource resource) throws RepositoryException {
+        synchronized (LOGGER) {
```

Comment:
If I comment out this `synchronized`, sometimes the new tests pass and sometimes they fail. If `synchronized` stays, I have not seen any test failures.
Maybe more concurrent threads would demonstrate the failure case more definitively.